Targets/Sample/Resources/Storyboards/02Collection/WorkspaceTableStoryboard.storyboard

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`NIT` Please separate `swift code` and `storyboard resource`. That could be more readable.

But this commit is simple now, so it's no problem. (Please consider this comment next time)